src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let string = '';
+
```

Comment:
I guess this name is bad, just string doesn't get us any understanding of what this string is created for

Reply:
agree